<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>janela_treino_usuario</class>
 <widget class="QMainWindow" name="janela_treino_usuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-color: #4A90E2;
}

QLabel{
	font-family: &quot;Segoe UI&quot;;
	font-size: 16px;
    color: #fff;
}

#_txt_treino{
	font-size: 26px;
	font-weight: bold;
}

#_txt_segunda{
	font-weight: bold;
}

#_txt_terca{
	font-weight: bold;
}

#_txt_quarta{
	font-weight: bold;
}

#_txt_quinta{
	font-weight: bold;
}

#_txt_sexta{
	font-weight: bold;
}

#_txt_comentario{
	font-weight: bold;
}

QPushButton{
    background: #4A90E2;
    border: 2px solid #fff;
    border-radius: 10px;
    padding: 10px 20px;

	font-family: &quot;Segoe UI&quot;;
	font-weight: bold;
	font-size: 18px;
    color: #fff;
}

QPushButton:hover{
    background: #53A3FF;
    border: 2px solid #fff;
    border-radius: 10px;
    padding: 10px 20px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="_txt_quinta">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Quinta:</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_quarta_user">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>500</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_comentario_user">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>500</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_segunda_user">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>500</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_sexta_user">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>500</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_treino">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Meu treino:</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_comentario">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Obsevações:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_voltar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Voltar</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_quinta_user">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>500</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_imagem">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="recursos.qrc">:/img/Imagens/imagem_bodybuilding.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_terca_user">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>500</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_sexta">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sexta:</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_quarta">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Quarta:</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_segunda">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Segunda:</string>
    </property>
   </widget>
   <widget class="QLabel" name="_txt_terca">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Terça:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="_btn_personal">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>500</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Personal</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections/>
</ui>
